dripline/implementations/entity_endpoints.py

```diff
@@ -0,0 +1,175 @@
+'''
+A Entity is an enhanced implementation of a Dripline Endpoint with simple logging capabilities.
+The Entitys defined here are more broad-ranging than a single service, obviating the need to define new Entitys for each new service or provider.
+
+When implementing a Entity, please remember:
+- All communication must be configured to return a response.  If no useful get is possible, consider a *OPC?
+- set_and_check is a generally desirable functionality
+
+Generic Entity catalog (in order of ease-of-use):
+- SimpleSCPIEntity: quick and simple minimal Entity
+- SimpleSCPIGetEntity/SimpleSCPISetEntity: limited instance of above with disabled Get/Set
+- FormatEntity: utility Entity with expanded functionality
+'''
+
+import asteval # used for FormatEntity, ADS1115CalcEntity
```

Comment:
ADS1115 stuff is purged?

Reply:
My initial feeling is that they probably belong in a plugin (maybe still in driplineorg) because they pull in extra dependencies and are pretty specialized. I think that model makes way more sense then the optional dependencies system.

Reply:
was just noting that the comment is outdated, but no impact on code

Reply:
oh, thanks